src/App.css

```diff
@@ -161,31 +203,96 @@ h2 {
     height: 400px;
   }
   .single-review {
-    display: flex;
-    flex-direction: row;
+    display: grid;
+    grid-template-areas:
+      "link link link link link link link"
+      "img img img img img text text"
+      "img img img img img text text"
+      "img img img img img text text"
+      ". comments comments comments comments comments ."
+      ". comments comments comments comments comments ."
+      ". comments comments comments comments comments ."
+      ". comments comments comments comments comments ."
+      ". comments comments comments comments comments .";
```

Comment:
This amount of columns and rows seems excessive. Seems like 3 rows and 2 columns would be sufficient